### ISSUE TITLE
Comparison version 1.8.1 vs 1.9

### ISSUE THREAD

**Issue by Raghothama1** [vdemichev/DiaNN#1041]
I searched DIA-NN versions 1.8.1 and 1.9 containing two replicates of DIA-MS data for the K562 digest. The Venn diagram shows more peptides found in the report.tsv file from version 1.9, but the number of protein groups in the report.pg_matrix is lower from version 1.9. Is version 1.9 more stringent?
![Compare_DIA-NN-18-1vs-1-9-](https://github.com/vdemichev/DiaNN/assets/146643293/11fb6f5d-660a-41ce-9cca-772aa601a95f)

**Comment by vdemichev** [comment]
Yes, it's more stringent, at both precursor & protein level, as well as the pg_matrix is in addition filtered at 5% run-specific protein group q-value.
